### DISCUSSION TITLE
Staking programs influence on tokenomics

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#93] [General]
by @RanTask in [Discord](https://discord.com/channels/906917846754418770/910524302531649566/977463892370395137):
> Guys, the new staking programs are great in addition to our LP reward program. However, I am a bit concerned about our tokenomics. As far as I understand the current rewards are paid out of existing reserves, respectively increasing of numbers of tokens in circularion.  I suggest we start to discuss a revenue model in order to finance future rewards. 

The Algostake program is completely financed by the Administration and Costs pool, both in sense of rewards ASASTATS tokens and STKE for the staking costs. The admpool has borrowed 250.00 ALGO from the devpool for the purpose.

 For the Yieldl'y staking programs, the admpool and the Airdrop pool [shared the costs in half](https://www.asastats.com/transparency-report-2022-02.pdf).

The first period of the [LP rewards incentive program](https://www.reddit.com/r/asastats/comments/slm2b4/asa_stats_is_officially_rolling_out_proposed_lp/) is funded by the airpool with an amount of 20,000,000 ASASTATS. It is expected that the rest of the required funds are funded by the admpool, the Rewards pool, and eventually the airpool.

https://www.asastats.com/whitepaper.pdf
> ASASTATS received by charging the professional services will be reallocated to the pools defined in the
Token allocation section. A prioritization in reallocating will take place based on the pools' current needs
and their states related to initial allocation.

**Comment by RanTask** [comment]
Thanks for the details. I am mainly concerned about that income stream. Do we already have an income from selling professional services?

**Reply by ipaleka** [reply]
> Thanks for the details. I am mainly concerned about that income stream. Do we already have an income from selling professional services?

Nope before we develop and start selling our API. But bear in mind that we won't exhaust the pools soon with a pace as we have got so far. 

Also, there's the DAO pool with more than a third of the total supply we can use. The whole purpose of the DAO pool is to cover critical situations and events and it is brought exactly for the purpose of staying on a long-term track without making hasty moves.
